### ISSUE TITLE
Unable to generate SOA record

### ISSUE THREAD

**Issue by pichouk** [elgs/dns-zonefile#23]
When I try to generate a zone file, it seems that the function `generate` does not generate SOA record.

I tried with a simplified version of the README :
```
let obj = {
	'$origin': 'MYDOMAIN.COM.',
	'$ttl': 3600,
	'soa': {
		'mname': 'NS1.NAMESERVER.NET.',
		'rname': 'HOSTMASTER.MYDOMAIN.COM.',
		'serial': '{time}',
		'refresh': 3600,
		'retry': 600,
		'expire': 604800,
		'minimum': 86400
	},
	'ns': [
      { 'host': 'NS1.NAMESERVER.NET.' },
      { 'host': 'NS2.NAMESERVER.NET.' }
	],
	'a': [
      { 'name': '@', 'ip': '127.0.0.1' },
      { 'name': 'www', 'ip': '127.0.0.1' },
      { 'name': 'mail', 'ip': '127.0.0.1' }
	]
};
console.log(zonefile.generate(obj));
```

It print this : 
```
; Zone: MYDOMAIN.COM.
; Exported  (yyyy-mm-ddThh:mm:ss.sssZ): 2017-03-27T07:41:35.426Z

$ORIGIN MYDOMAIN.COM.
$TTL 3600

; SOA Record
{name} {ttl}	IN	SOA	{mname}{rname}(
{serial} ;serial
{refresh} ;refresh
{retry} ;retry
{expire} ;expire
{minimum} ;minimum ttl
)

; NS Records
@	IN	NS	NS1.NAMESERVER.NET.
@	IN	NS	NS2.NAMESERVER.NET.

; MX Records

; A Records
@	IN	A	127.0.0.1
www	IN	A	127.0.0.1
mail	IN	A	127.0.0.1
```

I use the `0.2.0` version from NPM.

**Comment by elgs** [comment]
Thanks I will check that now.

**Comment by elgs** [comment]
@pichouk I confirm the horrible bug, thanks for reporting it. Now I have fixed it and have released `0.2.1` to the npm. Would you please try the new version and let me know if it has any problems. Thanks.

**Comment by pichouk** [comment]
Tried with bugfix in `0.2.1`, works very well. 
Thanks a lot ! (and +1 for reactivity :) )
